ui/src/components/UserList.tsx

```diff
@@ -16,32 +15,37 @@ type Props = {
 const UserList: React.FC<Props> = ({users, onAddFriend}) => {
   return (
     <List divided relaxed>
-      {[...users].sort((x, y) => x.username.localeCompare(y.username)).map((user) =>
-        <ListActionItem
-          key={user.username}
-          icon='user'
-          action={{
-            icon: 'add user',
-            onClick: () => onAddFriend(user.username),
-          }}
-          outer
-        >
-          <List.Header>{user.username}</List.Header>
+      {[...users].sort((x, y) => x.username.localeCompare(y.username)).map(user =>
+        <List.Item key={user.username}>
+          <List.Icon name='user' />
+          <List.Content>
+            <List.Content floated='right'>
+              <Icon
+                name='add user'
+                link
+                className='test-select-add-user-icon'
+                onClick={() => onAddFriend(user.username)} />
```

Comment:
```suggestion
                onClick={() => onAddFriend(user.username)}
              />
```
That's what the style guides seem to do.

Reply:
Fixed in https://github.com/digital-asset/create-daml-app/pull/72/commits/09b1942081baf0b50a7ff9db5c1ef4eb112157fe